docs/pages/enterprise/sso.mdx

```diff
@@ -199,12 +202,16 @@ spec:
 </TabItem>
 </Tabs>
 
+Note:
+- Manually pasting the XML to `entity_descriptor` field is error prone. We recommend using `tctl sso configure saml -e entity_descriptor.xml ...` command instead.
+- You may use `entity_descriptor_url`, in lieu of `entity_descriptor`, to fetch the entity descriptor from
+  your IDP. Though, we recommend "pinning" the entity descriptor by including the XML rather than fetching from a URL.
 
-You may use `entity_descriptor_url`, in lieu of `entity_descriptor`, to fetch
-the entity descriptor from your IDP. 
+Test the connector with [`tctl sso test`](../setup/reference/cli.mdx#tctl-sso-test) command:
```

Comment:
```suggestion
Test the connector with the `tctl sso test` command:
```
so we don't break the reader's flow

Reply:
I'm not entirely convinced that including the link constitutes the breaking of reader focus: we are not forcing them to follow the link, but rather making an option to do so if they are interested. This is similar to inline-link style in [Wikipedia](https://en.wikipedia.org/wiki/Teleport_(open-source_software)#Architecture), for example:

> == Architecture ==
> 
> Teleport is written in [Go](Go (programming language)) programming language, and runs on [UNIX](Unix)-compatible operating systems, including Linux, [macOS](macOS), and several [BSD/OS](BSD) variants. Teleport consists of two [executable](executable)s: <code>tsh</code> (command line client) and <code>teleport</code> (server daemon).

Sure, the reader can get sidetracked by clicking all the links, but my expectation is that they only do it when they want to get some clarification and would otherwise be lost.

Let me know what you think, though.

Reply:
I think with an inline link like this, the reader doesn't know what will be on the other side if they're not already familiar with the target URL. This means that a reader will need to open another browser window and shift context to make the determination of whether the link is useful. 

For a conceptual guide (e.g., a Wikipedia page), where a reader may already be browsing different sources of information, inline links are fine. For step-by-step guides, we're trying to keep all the relevant information on the same page. Otherwise, it can get frustrating to enter a tutorial thinking you'll achieve a goal by the end of it only to read incomplete instructions and run into roadblocks.

This guide is kind of a mix between a conceptual guide and a step-by-step guide. It has step-by-step instructions, but they're not comprehensive enough to follow as a tutorial. It covers a variety of topics conceptually, but not in much depth. The changes I'm suggesting make it more like a step-by-step guide.

Reply:
Ah, that makes sense, thank you for the explanation.